systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ slug|capfirst }} Systers
```

Comment:
Ok. So in this case you need to add to each template, that will be using this base template, the meetup location object (using the context). We have something similar with community object, which is added to the context using a mixin.

Also this hack won't work ok for meetup locations like "San Jose", they will become "San_jose".